definitions/ext-dora/definition.yml

```diff
@@ -0,0 +1,41 @@
+domain: EXT
+type: DORA
+synthesis:
+  rules:
+    - identifier: doraName
+      name: doraName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: DoraMetricsSample
+  tags:
+    doraName:
+      multiValue: false
+    repoName:
+      multiValue: false
+    deploymentFrequency:
+      multiValue: false
+    leadTimeForChanges:
+      multiValue: false
+    timeToRestoreService:
+      multiValue: false
+    changeFailureRate:
+      multiValue: false
```

Comment:
how often are these expected to change? Is like, every new datapoint will bring a new value in them?

Are these to display to the user? to filter?
They seem important as golden metrics but as tags it may put a lot of pressure on our side if they change too frequently.